mutateStrings/solution.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+function mutateMyStrings(stringOne, stringTwo){
+  let solution = [stringOne], s1 = stringOne.split(''), s2 = stringTwo.split('');
+
+  s1.forEach((char, index) => {if (char !== s2[index]) {

```

Comment:
It's convention and easier to read to put the 'if' statement and condition on the next line.
It makes deciphering the indentation of code blocks much easier.
Obviously, this code isn't that long so it's not an issue here, but with big code bases, having 'if' statements on the same line as the function before it can really make it hard to read. It's just a good habit to get into.